workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-capability-cards/ddm-template.ftl

```diff
@@ -0,0 +1,71 @@
+<#assign taxonomyVocabularies = restClient.get("/headless-admin-taxonomy/v1.0/sites/${themeDisplay.getSiteGroupId()}/taxonomy-vocabularies").items />
+
+<#if taxonomyVocabularies?has_content>
+	<#list taxonomyVocabularies as taxonomyVocabulary>
+		<#if stringUtil.equals(taxonomyVocabulary.externalReferenceCode, "CAPABILITY")>
+			<#assign capabilityId = taxonomyVocabulary.id />
+		</#if>
+	</#list>
+
+	<div class="col-12 m-0 product-cards row">
+		<#list restClient.get("/headless-admin-taxonomy/v1.0/taxonomy-vocabularies/${capabilityId}/taxonomy-categories").items?sort_by("name") as taxonomyCategory>
+			<div class="card-container col-12 col-md-4 col-sm-6 d-flex justify-content-center p-2">
+				<a class="d-flex home-card p-3" href="/search?category=${taxonomyCategory.id}">
+					<#if taxonomyCategory.taxonomyCategoryProperties?has_content>
+						<#list taxonomyCategory.taxonomyCategoryProperties as taxonomyCategoryProperty>
+							<#if taxonomyCategoryProperty.key == 'icon'>
+								<img
+									alt="${taxonomyCategory.name} icon"
+									class="icon mr-3"
+									src="${taxonomyCategoryProperty.value}"
+								/>
+							</#if>
+						</#list>
+					</#if>
+
+					<div>
+						<h6 class="title">
+							${taxonomyCategory.name}
+						</h6>
+
+						<p class="pt-2 subtitle">
+							${taxonomyCategory.description}
+						</p>
+					</div>
+				</a>
+			</div>
+		</#list>
+	</div>
+</#if>
+
+<style>
```

Comment:
move the style to the start of the article

Reply:
Done.